styles/Card.module.css

```diff
@@ -0,0 +1,79 @@
+.container {
+    background-color: transparent;
+    color: var(--white);
+    text-align: center;
+    position: relative;
+    cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 10% 0;
+    align-items: center;
+    border-radius: 1%;
+}
+
+.profile_img {
+    height: 280px;
+    width: 70%;
+    object-fit: cover;
+    background-size: cover;
+    background-image: url(https://images.unsplash.com/photo-1497316730643-415fac54a2af?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1900&q=80);
```

Comment:
Use background-image in inline styling or use next/Image tag as we have to pass images as props